test/structure/test_atom.py

```diff
@@ -49,90 +53,134 @@ def test_element_canonical():
     assert len(atom3.get_connect()) == 3
     assert "missing connecting atom " in caplog.text
 
+    enzy_htp._LOGGER.setLevel(original_level)
+
+
 def test_get_connect_w_h():
     """get connect with no missing H atoms"""
     stru = PDBParser().get_structure(f"{DATA_DIR}1Q4T_peptide_protonated.pdb")
-    atom1 = stru["A"][0][0] #N (Nter)
-    atom2 = stru["A"][1][0] #N
-    atom3 = stru["A"][-1].find_atom_name("C")#N (Cter)
+    atom1 = stru["A"][0][0]  #N (Nter)
+    atom2 = stru["A"][1][0]  #N
+    atom3 = stru["A"][-1].find_atom_name("C")  #N (Cter)
     assert len(atom1.get_connect()) == 4
     assert len(atom2.get_connect()) == 3
     assert len(atom3.get_connect()) == 3
 
+
 def test_attached_protons():
     """test if the function gives correct attached protons"""
     stru = PDBParser().get_structure(f"{DATA_DIR}1Q4T_peptide_protonated.pdb")
-    atom1 = stru["A"][0][0] #N (Nter)
-    assert list(map(lambda a: a.name, atom1.attached_protons())) == ['H1', 'H2', 'H3']
+    atom1 = stru["A"][0][0]  #N (Nter)
+    assert list(map(lambda a: a.name,
+                    atom1.attached_protons())) == ['H1', 'H2', 'H3']
+
 
-# TODO recover when need
-@pytest.mark.TODO
 def test_distance():
-    """Making sure that the .distance() method works."""
-    TEST_PDB_NAME = f"{CURR_DIR}/data/atom_test_pdb.pdb"
-    RAW_ATOMS: pd.DataFrame = PandasPdb().read_pdb(TEST_PDB_NAME).df["ATOM"]
-    atoms: List[Atom] = list(map(lambda pr: Atom(**pr[1]), RAW_ATOMS.iterrows()))
-    atom1 = Atom(x_coord=0, y_coord=0, z_coord=0)
-    atom2 = Atom(x_coord=0, y_coord=0, z_coord=1)
-    assert np.isclose(atom1.distance(atom2), 1)
-    assert np.isclose(atom2.distance(atom1), 1)
-
-    atom1 = Atom(x_coord=0, y_coord=0, z_coord=0)
-    atom2 = Atom(x_coord=0, y_coord=1, z_coord=0)
-    assert np.isclose(atom1.distance(atom2), 1)
-    assert np.isclose(atom2.distance(atom1), 1)
-
-    atom1 = Atom(x_coord=0, y_coord=0, z_coord=0)
-    atom2 = Atom(x_coord=1, y_coord=0, z_coord=0)
-    assert np.isclose(atom1.distance(atom2), 1)
-    assert np.isclose(atom2.distance(atom1), 1)
-
-    atom1 = Atom(x_coord=0, y_coord=0, z_coord=0)
-    atom2 = Atom(x_coord=-1, y_coord=0, z_coord=0)
-    assert np.isclose(atom1.distance(atom2), 1)
-    assert np.isclose(atom2.distance(atom1), 1)
-
-    atom1 = Atom(x_coord=0, y_coord=0, z_coord=0)
-    atom2 = Atom(x_coord=0, y_coord=-1, z_coord=0)
-    assert np.isclose(atom1.distance(atom2), 1)
-    assert np.isclose(atom2.distance(atom1), 1)
-
-    atom1 = Atom(x_coord=0, y_coord=0, z_coord=0)
-    atom2 = Atom(x_coord=0, y_coord=0, z_coord=-1)
-    assert np.isclose(atom1.distance(atom2), 1)
-    assert np.isclose(atom2.distance(atom1), 1)
-
-    atom1 = Atom(x_coord=0, y_coord=0, z_coord=0)
-    atom2 = Atom(x_coord=0, y_coord=0, z_coord=0)
-    assert np.isclose(atom1.distance(atom2), 0)
-    assert np.isclose(atom2.distance(atom1), 0)
+    """Making sure that the .distance_to() method works."""
```

Comment:
good fix!

Reply:
Be great if you can also change the name of the test to test_distance_to

Reply:
Fixed.